frontend/js/src/utils/utils.tsx

```diff
@@ -467,6 +468,16 @@ const getPageProps = (): {
       sentry_dsn,
     } = globalReactProps;
 
+    let { user_preferences } = globalReactProps;
+
+    user_preferences = { ...user_preferences, saveData: true };
```

Comment:
I don't think this should be true by default. Did you mean it to be this instead?
```suggestion
    user_preferences = { ...user_preferences, saveData: false };
```

Reply:
Yeah, this should've been false by default! Thanks for that